ui/00_message/pawn/p0001.xml

```diff
@@ -17,22 +19,28 @@
   </message>
   <message>
     <original>冒険の手助けなら、いつでもいたします</original>
+	<translation>If you are in need of help on your adventure, we are here for you.</translation>
```

Comment:
Another nitpick, though not a necessary call for a change, you use "we" a lot for the pawn, but it's not necessarily true that there will be more than one pawn present. I believe most of these should be "I".